UKHomeOffice/nmsw-ui
# NMSW 1374

# Ticket
JIRA-1374

----

## AC
 - User should log in and be presented with ETA (interstitial) page
 - User should log out and back in and not be presented with ETA (interstitial) page

----

## To test
 - Run development server
 - Log in, you should see the ETA (interstitial) page
 - Log out
 - Log in, you should NOT see the ETA (interstitial) page

----

## Notes
New tests written which should also pass
